### ISSUE TITLE
Displays a message about api-ms-win-crt-runtime-l1-1-0.dll being missing from your computer at start up

### ISSUE THREAD

**Issue by TafThorne** [simonsteele/pn#102]
When I try to start Programmers Notepage I am presented with a pn.exe - System Error message stating that "The program can't start because pn.exe api-ms-win-crt-runtime-l1-1-0.dll is missing from your computer. Try reinstalling the program to fix this problem." 

When I click the OK button the application seems to finished loading successfully.  

Reinstalling Version 2.3 does not resolve the issue.  
Installing version 2.4.2378-duke does not resolve the issue.

**Comment by simonsteele** [comment]
Did this start when you installed 2.4?

If so, did you install the PyPN plugin?

This sounds like an error being shown when one of the plugins is loaded, as the app runs afterwards.

**Comment by TafThorne** [comment]
>Did this start when you installed 2.4?
No, I only installed 2.4 to try and fix the issue.  I was on 2.3 when it began.  

>If so, did you install the PyPN plugin?
I do not recall installing any plugins.  I can try to check...
Are plugins the things that turn up under Options > Extensions?  If so I only have the CTags Tagger v1.6 and Customer Scheme Lexer v2.0 installed.  

Strangely the problem did not occur just now, despite happening for months on this PC.  Is the missing DLL something that Windows update could have dragged in?  I have installed Ekahau, Git & Freemind recently too so one of them could have brought the missing file with them I suppose.
